README.md

```diff
@@ -1 +1,35 @@
-# paas-accounts
\ No newline at end of file
+# paas-accounts
+
+API for storing information about PaaS users.
+
+## Documents
+
+### PUT /documents/:name
+
+Create or update a document:
+
+    curl -u <USER>:<PASS> -H "Content-Type: application/json" -X PUT -d '{"content": "my content"}' https://<HOSTNAME>/documents/my_document
+
+### GET /documents/:name
+
+Retrieve an existing document:
+
+    curl -u <USER>:<PASS> https://<HOSTNAME>/documents/my_document
+
+## Agreements
+
+### POST /agreements
+
+Record a user's agreement to a document:
+
+    curl -u <USER>:<PASS> -H "Content-Type: application/json" -X POST -d '{"user_uuid": "00000000-0000-0000-0000-000000000001", "document_name": "my_document"}' https://<HOSTNAME>/agreements
+
+### GET /users/:uuid/documents
+
+Get all documents for a user:
+
+    curl -u <USER>:<PASS> https://<HOSTNAME>/users/00000000-0000-0000-0000-000000000001
```

Comment:
Should this be `curl -u <USER>:<PASS> https://<HOSTNAME>/users/00000000-0000-0000-0000-000000000001/documents`?